src/io/RotatedProjWriter.cpp

```diff
@@ -66,13 +72,13 @@ void io::RotatedProjWriter::operator()(Grid3D &G, Parameters P, int nfile, const
   // create the filename
   std::string filename = fname_template.format_fname(nfile, "_rot_proj");
 
+  // it may be a little more explicit to use a switch statement instead of if/elif/else
```

Comment:
I agree that a switch statement would be clearer if you wanted to change it (but I don't think it's necessary for this PR if you don't feel like it).